cocoapods/CLOpenSSL.podspec.template

```diff
@@ -0,0 +1,214 @@
+Pod::Spec.new do |s|
+    # These parts of the spec are filled in by the template generator.
+    openssl_version = "%%OPENSSL_VERSION%%"
+    min_target_ios  = "%%MIN_IOS_SDK%%"
+    min_target_osx  = "%%MIN_OSX_SDK%%"
+
+    github_repo = "%%GITHUB_REPO%%"
+    iPhone_archive_name = "%%IPHONE_ARCHIVE_NAME%%"
+    iPhone_archive_hash = "%%IPHONE_ARCHIVE_HASH%%"
+    macOSX_archive_name = "%%MACOSX_ARCHIVE_NAME%%"
+    macOSX_archive_hash = "%%MACOSX_ARCHIVE_HASH%%"
+
+    # Project metadata
+    s.name     = "CLOpenSSL"
+    s.version  = "#{openssl_version}"
+    s.summary  = "Robust, commercial-grade, and full-featured toolkit for the TLS and SSL protocols as well as a general-purpose cryptography library."
```

Comment:
```suggestion
    s.summary  = "OpenSSL pre-built framework for iOS and macOS. Robust, commercial-grade, and full-featured toolkit for the TLS and SSL protocols as well as a general-purpose cryptography library."
```

Reply:
Well, [CocoaPods guidelines](https://guides.cocoapods.org/syntax/podspec.html#summary) suggest that the summary should be short, and there is no need to specify that it's a framework (every pod is a library/framework), or that it's OpenSSL (which obvious from the name).

If you insist, how about this one which is still one sentence?
```suggestion
    s.summary  = "Pre-built OpenSSL framework for iOS and macOS: robust, commercial-grade, and full-featured toolkit for the TLS and SSL protocols as well as a general-purpose cryptography library."
```

Reply:
Well, my primary idea was to clarify for users why there're different OpenSSL CocoaPods.